onadata/libs/utils/csv_builder.py

```diff
@@ -383,7 +383,14 @@ def __init__(self, username, id_string, filter_query=None,
                  include_images=False, include_hxl=False,
                  win_excel_utf8=False, total_records=None,
                  index_tags=DEFAULT_INDEX_TAGS, value_select_multiples=False,
-                 show_choice_labels=False, language=None):
+                 show_choice_labels=False, language=None,
+                 include_review=False):
+
+        if include_review:
+            if not all(
+                elem in self.ADDITIONAL_COLUMNS for elem in [REVIEW_STATUS,
```

Comment:
Are there situations where the review fields would have been included?

Reply:
I found this happening when I running the tests so I implemented the fix. I have realized the two fields are being added after every test that has the option include_review set to true. Am working on fixing this